src/dotnet/install.sh

```diff
@@ -4,44 +4,10 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 #
-# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/dotnet.md
-# Maintainer: The VS Code and Codespaces Teams
-
-DOTNET_VERSION="${VERSION:-"latest"}"
-DOTNET_RUNTIME_ONLY="${RUNTIMEONLY:-"false"}"
-OVERRIDE_DEFAULT_VERSION="${OVERRIDEDEFAULTVERSION:-"true"}"
-INSTALL_USING_APT="${INSTALLUSINGAPT:-"true"}"
-
-DOTNET_LATEST="7"
-DOTNET_LTS="6"
-
-USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
-UPDATE_RC="${UPDATE_RC:-"true"}"
-TARGET_DOTNET_ROOT="${TARGET_DOTNET_ROOT:-"/usr/local/dotnet"}"
-ACCESS_GROUP="${ACCESS_GROUP:-"dotnet"}"
-
-MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
-DOTNET_ARCHIVE_ARCHITECTURES="amd64"
-DOTNET_ARCHIVE_VERSION_CODENAMES="buster bullseye bionic focal hirsute jammy"
-# Feed URI sourced from the official dotnet-install.sh
-# https://github.com/dotnet/install-scripts/blob/1b98b94a6f6d81cc4845eb88e0195fac67caa0a6/src/dotnet-install.sh#L1342-L1343
-DOTNET_CDN_FEED_URI="https://dotnetcli.azureedge.net"
-# Ubuntu 22.04 and on do not ship with libssl1.1, which is required for versions of .NET < 6.0
-DOTNET_VERSION_CODENAMES_REQUIRE_OLDER_LIBSSL_1="buster bullseye bionic focal hirsute"
-
-# Comma-separated list of dotnet versions to be installed
-# alongside DOTNET_VERSION, but not set as default.
-ADDITIONAL_VERSIONS=${ADDITIONALVERSIONS:-""}
```

Comment:
Can we add this back? It makes the variable readable.

From earlier conversations, I understand the reason behind this change https://github.com/devcontainers/features/commit/2b2e683da219ca23b892c49376b0449aeb38af84. Can we update it to 👇  instead?

```
ADDITIONAL_VERSIONS=${ADDITIONALVERSIONS}
```

Reply:
Done